src/js/mqtt.js

```diff
@@ -340,7 +349,7 @@ MqttClient.prototype.unsubscribe = function(topic, callback) {
   // TODO don't use try catch
   try {
     buf = this._handle._getUnsubscribe(topic, {
-      id: this._msgId++,
+      id: ++this._msgId,
```

Comment:
QoS is missing here?

Reply:
There is not `options` argument here.

Reply:
> There is not `options` argument here.

What I mean is that if the unsubscribe need qos option?

Reply:
there is not `qos` option in unsubscribe.